ecologits/electricity_mix_repository.py

```diff
@@ -3,6 +3,7 @@
 from dataclasses import dataclass
 from typing import Optional
 
+reserved_codes = ["WOR", "EEE", "SCG"]
```

Comment:
@samuelrince I based the tool on the alpha-3 codes, but there are some exceptions. Some codes are "reserved", but not officially country ISO codes. In the ADEME database, 3 of them are such codes, namely "WOR" (reserved for world), "EEE" (reserved for Europe) and "SCG" (reserved for Serbia and Montenegro). 

When I check with pycountry that the user uses a valid ISO code in `tracers/utils.py`, these codes are valid exceptions